<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheminfoDatabaseWorkPanel</class>
 <widget class="QWidget" name="CheminfoDatabaseWorkPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Database panel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/CheminfoProject/12.cheminfo/Database.svg</normaloff>:/images/CheminfoProject/12.cheminfo/Database.svg</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton {
     border: 1px solid #dadbde;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
     border-radius: 2px;
     min-width: 34px;
     min-height: 24px;
 }

 QPushButton:checked {
     border: 2px solid #777777;
 }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabsDatabase">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabStatus">
        <attribute name="title">
         <string>Status</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>250</width>
           <height>100</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Database state:</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="labelConnect">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>101</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Disconnected</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="connectionPB">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>79</y>
           <width>261</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string>Structure database state:</string>
         </property>
         <widget class="QLabel" name="labelConnectStructure">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>101</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Not found</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="browseDBStructurePB">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>149</y>
           <width>261</width>
           <height>101</height>
          </rect>
         </property>
         <property name="title">
          <string>Project receptor:</string>
         </property>
         <widget class="QLineEdit" name="receptorLE">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>221</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="browseReceptorPB">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>60</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tabSettings">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <widget class="QGroupBox" name="databaseSet">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>291</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Database:</string>
         </property>
         <widget class="QPushButton" name="dbApplyPB">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>210</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>281</width>
            <height>184</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="labelDBName">
             <property name="text">
              <string>DB name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="dbPathLE"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelDBPath">
             <property name="text">
              <string>DB path:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="dbNameLE"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="dbUserLE"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="dbPassLE">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelDBPass">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelDBUser">
             <property name="text">
              <string>User:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>DB type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dbTypeCB">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Firebird</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="saveDatabaseCB">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Save settings into the file?
(in next startup will be setting restore from the settings file)</string>
          </property>
          <property name="text">
           <string>Save settings</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="structureDatabaseSet">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>291</width>
           <height>231</height>
          </rect>
         </property>
         <property name="title">
          <string>Structure database:</string>
         </property>
         <widget class="QPushButton" name="dbStructApplyPB">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>200</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>281</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QLabel" name="ligTypeLabel">
             <property name="text">
              <string>Ligands type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelstructPath">
             <property name="text">
              <string>Path:          </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="structPrefixLE"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="structPathLE"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelstructPrefix">
             <property name="text">
              <string>Prefix:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="ligTypeCB"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="resTypeLabel">
             <property name="text">
              <string>Results type:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="resTypeCB"/>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="hierarchyCB">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>170</y>
            <width>97</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Hierarchy</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="saveDatabaseStructureCB">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>131</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Save settings into the file?
(in next startup will be setting restore from the settings file)</string>
          </property>
          <property name="text">
           <string>Save settings</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>connectionPB</tabstop>
  <tabstop>browseDBStructurePB</tabstop>
  <tabstop>tabsDatabase</tabstop>
  <tabstop>dbNameLE</tabstop>
  <tabstop>dbPathLE</tabstop>
  <tabstop>dbUserLE</tabstop>
  <tabstop>dbPassLE</tabstop>
  <tabstop>dbApplyPB</tabstop>
  <tabstop>structPathLE</tabstop>
  <tabstop>dbStructApplyPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
